crates/admin/src/service.rs

```diff
@@ -107,6 +110,55 @@ where
             self.metadata_writer.raw_metadata_store_client(),
         ));
 
+        // Add now the tracing layer, this makes sure we don't log ui requests
+        let router = router.layer(
+            TraceLayer::new_for_http()
+                .make_span_with(|request: &Request<_>| {
+                    info_span!(
+                        target: "restate_admin::api",
+                        "admin-api-request",
+                        http.version = ?request.version(),
+                        http.request.method = %request.method(),
+                        url.path = request.uri().path(),
+                        url.query = request.uri().query().unwrap_or_default(),
+                        url.scheme = request.uri().scheme_str().unwrap_or("http")
+                    )
+                })
+                // Just log on response
+                .on_request(())
```

Comment:
If this is intended to be an audit log, I think we must rather log on request - we would prefer to log the access even if it didn't happen over even partially processing the request without logging.

Reply:
I was mimicking what apache does: https://httpd.apache.org/docs/2.4/logs.html and it seems it logs on response...